assets/xml/objects/object_link_child.xml

```diff
@@ -0,0 +1,227 @@
+<Root>
+    <File Name="object_link_child" Segment="6">
+        <Skeleton Name="gLinkChildSkel" Type="Flex" LimbType="LOD" Offset="0x2CF6C"/>
+        
+        <!--Far Limb DLists--> 
+        <DList Name="gChildLimbLeftHandFarDL" Offset="0x16280"/>   
```

Comment:
The naming in this file seems very inconsistent. Some variables are named with "ChildLink", some with "LinkChild" and even some with just "Child". I think it would be nice to get at least a consistent scheme for this.